notebooks/tel_and_site/subsys_req_ver/m1m3/LVV-T232-analysis_actuators.ipynb

```diff
@@ -0,0 +1,397 @@
+{
```

Comment:
<p>This might be a good candidate of being an external function. The other option would be to use the internal matplotlib's animation (here is an <a href="here is an [example](https://matplotlib.org/stable/gallery/animation/simple_anim.html" target="_blank">example</a>). Not critical. We can also leave this as it is right now. Just throwing out ideas. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>